### ISSUE TITLE
to is null

### ISSUE THREAD

**Issue by zjb0807** [AcalaNetwork/bodhi.js#477]
evmProvider.getTransactionByHash(txhash);
```
{
  blockHash: '0xac9cd9b6efd49b65fc7cb630f9a2f582fd406e522aea0431feb994ff47c77936',
  blockNumber: 145,
  transactionIndex: 0,
  hash: '0x9d0ca2ae823b999d35052aae77eefdcdf70ec3ea930c767faf7813060624e6fa',
  from: '0x82A258cb20E2ADB4788153cd5eb5839615EcE9a0',
  gasPrice: BigNumber { _hex: '0x8acb0566f1', _isBigNumber: true },
  value: '0x',
  gas: 2100000,
  input: '0x',
  to: null,
  nonce: 141,
  v: '0x25',
  r: '0x1b5e176d927f8e9ab405058b2d2457392da3e20f328b16ddabcebc33eaac5fea',
  s: '0x4ba69724e8f69de52f0125ad8b3c5c2cef33019bac3249e2c0a2192766d1721c'
}
```

**Comment by zjb0807** [comment]
It happens in #466

**Comment by zjb0807** [comment]
Closed by #479
